include/jsonbuilder/JsonBuilder.h

```diff
@@ -419,21 +478,23 @@ nodes.) To add new types in your own project:
   and jsonValue.ConvertTo<T> to work for your types,
   define specializations of JsonImplementType<T> for each of your types.
 */
-enum JsonType : uint8_t
+enum JsonType
+    : JsonInternal::JSON_UINT32 // Note: Used as an 8-bit bitfield of UINT32.
 {
     // Numbering for custom types should start at 1. Custom types never have
     // children. Numbering for custom types must not exceed 200.
     JsonTypeReserved = 201,
     JsonTypeBuiltIn = 244,
-    JsonUtf8,    // No children. Data = utf-8 string.
-    JsonUInt,    // No children. Data = uint (1, 2, 4, or 8 bytes).
-    JsonInt,     // No children. Data = int (1, 2, 4, or 8 bytes).
-    JsonFloat,   // No children. Data = float (4 or 8 bytes).
-    JsonBool,    // No children. Data = bool (1 or 4 bytes).
-    JsonTime,    // No children. Data = uint64 (number of 100ns intervals since
-                 // 1601-01-01T00:00:00Z, i.e. Win32 FILETIME).
+    JsonUtf8,    // No children. Data = UTF-8 string.
+    JsonUInt,    // No children. Data = uint (1, 2, 4, or 8 bytes, little-endian).
+    JsonInt,     // No children. Data = int (1, 2, 4, or 8 bytes, little-endian).
+    JsonFloat,   // No children. Data = float (4 or 8 bytes, little-endian).
+    JsonBool,    // No children. Data = bool (1 or 4 bytes, little-endian).
+    JsonTime,    // No children. Data = int64 (number of 100ns intervals since
+                 // 1601-01-01T00:00:00Z, compatible with Win32 FILETIME).
     JsonUuid,    // No children. Data = UUID (16 bytes universally unique
-                 // identifier, i.e. Win32 GUID).
+                 // identifier, network byte order, compatible with uuid_t,
+                 // NOT the same byte order as Windows GUID).
```

Comment:
Note: Explicitly define value layout since layout needs to be consistent across machines.